test/functional/cmdline_options_tester/src/Tokenizer.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2013, 2018 IBM Corp. and others
+ * Copyright (c) %s, %s IBM Corp. and others
```

Comment:
The first date should not be changed (it should remain 2013); the second date should be 2020 (or soon 2021).